web/public/js/directives/cd-eventbrite-integration.js

```diff
@@ -0,0 +1,76 @@
+;(function() {
+  'use strict';
+
+function cdEventbriteIntegration($window, cdEventbriteService, alertService, atomicNotifyService,
+  $localStorage, $stateParams, $state, $translate, $rootScope) {
+    return {
+      restrict: 'E',
+      template:
+        '<i class="fa fa-spinner" ng-show="saving"></i>' +
```

Comment:
Sorry, I just meant put the template in its own file instead of an inline template. Inline templates that are more than a single line make me sad 😢